doc/classes/VisualShaderNodeWorldPositionFromDepth.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="VisualShaderNodeWorldPositionFromDepth" inherits="VisualShaderNode" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A visual shader node that calculates the position of the pixel in world space using the depth texture.
+	</brief_description>
+	<description>
```

Comment:
Still missing description.
It's for completeness sake 🙃

Reply:
In that Case the brief description already tells everything the Node does. Should I just copy the text? :/

Reply:
idk, maybe mention the port types? Or where can it be used? (I assume only in spatial shaders)